PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -121,6 +121,18 @@ public class PocketSource: Source {
             completion()
         }
     }
+
+    public func saveFromExtension(url: URL) {
+        let x: ExtensionSaveRequest = space.new()
+        x.url = url
+
+        try! space.save()
+    }
+
+    public func checkForNewThings() {
+        let saveRequests = try! space.context.fetch(ExtensionSaveRequest.fetchRequest())
```

Comment:
This gets called when the app enters the foreground. If any items have been saved via the extension, this query should return them. At this point, we can enqueue any operations necessary.